src/test/resources/fixtures/ifs/if-avoid-not-operator.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+if (<weak_warning descr="This negative if conditional could be avoided">!expr()</weak_warning>) { a(); b(); }
+else { c(); d(); }
+
+if (!true) { a(); b(); }
```

Comment:
I have tried on vendor folder on a Laravel project and I have found a lot of compatible cases,  none false positive. :)